backend/lib/controllers/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * This file is part of the ALICE Electronic Logbook v2, also known as Jiskefet.
+ * Copyright (C) 2020  Stichting Hogeschool van Amsterdam
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+const HomeController = require('./home.controller');
+const UserController = require('./users.controller');
```

Comment:
```suggestion
const UsersController = require('./users.controller');
```

Reply:
As far as I could see controllers should not be plural: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes

Reply:
Their endpoints match the name of the controller, thus /book/X goes to the BookController. So this might be question of whether or not the routes/controllers should be plural or singular and match each other (see /users).